products/platform/documentation-site/research/search-design-doc.md

```diff
@@ -12,32 +12,75 @@
 
 ### Background
 
 The documentation we have is spread across many different locations (public repos, private repos, GitHub pages, handbooks, etc). There currently isn't a way for VFS/VSP users to search multiple documentation sources with a single query.
 
 ### High Level Design
 
-We are adding a public landing page that will contain a search input where users can search multiple sources of documentation with one query.
-
 ## Specifics
 
 ### Detailed Design
 
-We are adding a public landing page that will contain the [DocSearch frontend library](https://github.com/algolia/docsearch), and running our own instance of the [`docsearch-scraper`](https://github.com/algolia/docsearch-scraper) to extract content from our documentation sources and push it to an Algolia index.
+For a multi-repo search MVP, we are adding an HTML text input and [Algolia's DocSearch JS snippet](https://github.com/algolia/docsearch) to the [`va.gov-team` GitHub Pages site](https://department-of-veterans-affairs.github.io/va.gov-team/). This will use [Algolia's Search API](https://www.algolia.com/products/search/) to query Algolia's hosted database (https://www.algolia.com/pricing/), which will be populated by [Algolia's scraper](https://github.com/algolia/docsearch-scraper) using a [configuration we supply](docs/docsearch-scraper-config.json).
+
+In general, this kind of system requires the following components:
+
+- Page with Input Textbox - We need a public page with a search input where users can input their search term(s).
+- Index/Database - We need a place to store the records that will be returned to the user.
+- Search API - We need a way to get data out of the index.
+- Crawler/Scraper - We need a crawler to scan our documentation sources and push data to our index.
+- Pipeline - We need a way to routinely run the crawler
+
+By using Algolia's Search API product with their open source Dropdown Search-UI and open source scraper, we can avoid building/maintaining the database, API, UI, and crawler components.
+
+That means we'd be responsible for:
+
+- Converting `va.gov-team`'s `docs/index.md` from markdown to HTML
+- Adding a search input to `va.gov-team`'s `docs/index.html`
+- Adding the DocSearch `<style>` and `<script>` tags to `va.gov-team`'s `docs/index.html`
+- Writing the config file for the scraper
+- Adding CircleCI to `va.gov-team`
+- Adding a scheduled job to `va.gov-team`'s CircleCI pipeline to run the scraper once every 24 hours
+- Manually triggering the initial scraper run to populate our Algolia index
+
+#### Usage limits
+
+[Algolia's pricing page](https://www.algolia.com/pricing/) shows three different tiers, in addition to an enterprise plan. The primary differentiator between the tiers is the number of records and the number of operations (read/write/edit/delete) included with each plan.
+
+##### The Community Plan
```

Comment:
which of these plans is most relevant to your design? which one do you think we'll need to procure?

Reply:
Short answer: the Pro plan

I've updated this section, and included my calculations, in https://github.com/department-of-veterans-affairs/va.gov-team/pull/5248/commits/18b1e99e326c38006711b90b8754dfc49658f103.